tests/integration.test.ts

```diff
@@ -0,0 +1,34 @@
+import { OMGCLI } from "../src/omgcli";
+
+const config = require("../config.js");
+
+jest.setTimeout(200000);
+
+let omgcli: OMGCLI;
+//let processingUTXOPos: number[] = [];
+
+beforeEach(() => {
+  omgcli = new OMGCLI(config);
+  // processingUTXOPos = [];
+});
+
+test("Get UTXOs for an address", async () => {
+  const response = await omgcli.getUTXOs(config.alice_eth_address);
+
+  expect(response).toBeInstanceOf(Array);
```

Comment:
Not a blocking comment but should we be testing the contents of this array? 

Could use a UTXO interface and have `const response:UTXO[] = await ...` 

```ts
export interface UTXO {
  amount: string
  blknum: number
  creating_txhash: string | null
  currency: string
  inserted_at: string
  oindex: number
  otype: number
  owner: string
  spending_txhash: string | null
  txindex: number
  updated_at: string
  utxo_pos: number
}
```

Reply:
Normally I would say absolutely yes but for this tool I think it might be overkill. The purpose of it is to test and to leave things relatively dynamic. I would not want it to break just because some attribute has been added or the user does not have any utxos yet.